eyeshade/controllers/owners.js

```diff
@@ -44,14 +44,15 @@ v1.bulk = {
     return async (request, reply) => {
       const authorizer = request.payload.authorizer
       const providers = request.payload.providers || []
-      let extras
+      const channels = []
+      let info = {}
 
-      if (authorizer.ownerEmail || authorizer.ownerName) {
-        extras = underscore.omit(authorizer, [ 'owner' ])
-        providers.forEach((provider) => { underscore.extend(provider, extras) })
-      }
+      if (authorizer.ownerEmail || authorizer.ownerName) info = { email: authorizer.ownerEmail, name: authorizer.ownerName }
+      providers.forEach((provider) => {
+        channels.push(underscore.extend({ channeId: provider.publisher, visible: provider.show_verification_status }, info))
```

Comment:
`channeId` - typo?

Reply:
not a typo. the folks at tilde requested that we use `channelId` for the `/v2/...` endpoint. since this is the `/v1/...` endpoint and they both use the same worker routine, we convert it.